src/Apps/Artist/ArtistApp.tsx

```diff
@@ -1,14 +1,15 @@
 import { Col, Row, Separator, Spacer } from "@artsy/palette"
 import { ArtistApp_artist } from "__generated__/ArtistApp_artist.graphql"
 import { NavigationTabsFragmentContainer as NavigationTabs } from "Apps/Artist/Components/NavigationTabs"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
```

Comment:
These two components are used to frame new apps seen in the `/Apps` folder. In the future these can be combined into something like `<Layout>`